src/uct/ib/base/ib_device.c

```diff
@@ -1132,7 +1132,10 @@ static ucs_sys_device_t uct_ib_device_get_sys_dev(uct_ib_device_t *dev)
         return UCS_SYS_DEVICE_ID_UNKNOWN;
     }
 
-    ucs_topo_sys_device_set_name(sys_dev, uct_ib_device_name(dev));
+    status = ucs_topo_sys_device_set_name(sys_dev, uct_ib_device_name(dev));
+    if (status != UCS_OK) {
+        return UCS_SYS_DEVICE_ID_UNKNOWN;
```

Comment:
i think we can assert status is OK, because we just got sys_dev back from ucs_topo_find_device_by_bus_id()

Reply:
done